tap_looker/transform.py

```diff
@@ -81,15 +85,15 @@ def tranform_looker_schemas(this_dict):
 
 # Create a Singer JSON Schema from Looker Swagger endpoint
 def get_transform_schema(client, stream_swagger_object, stream_name):
-    LOGGER.info('Starting transform: %s', stream_name)
+    LOGGER.info('Starting transform: {}'.format(stream_name))
```

Comment:
`LOGGER.info('Starting transform: %s', stream_name)` is the way this should be done.

Reply:
Ok - changed.